server/snapshot/snapshot_test.go

```diff
@@ -46,60 +47,68 @@ func TestGetSnapshotKeyCreate(t *testing.T) {
 
 	require.Nil(t, err, "Expected nil error")
 
-	// trying to get the same key again should return the same value
-	require.Equal(t, k, k2, "Did not receive same key when attempting to recreate.")
+	// trying to get the key for the same gun and role will create a new key unless we have existing TUF metadata
```

Comment:
Could we also add a test for if there is existing TUF metadata, but the key in the metadata is not held by the signer?  (similarly for timestamp?)